hack/multikueue-e2e-test.sh

```diff
@@ -87,16 +85,7 @@ function kind_load {
     # In order for MPI-operator and Training-operator to work on the same cluster it is required that:
     # 1. 'kubeflow.org_mpijobs.yaml' is removed from base/crds/kustomization.yaml - https://github.com/kubeflow/training-operator/issues/1930
     # 2. Training-operator deployment is modified to enable all kubeflow jobs except for mpi -  https://github.com/kubeflow/training-operator/issues/1777
-   
-    # Modify the `newTag` for the `kubeflow/training-operator` to use the one training-operator version
-    $YQ eval '(.images[] | select(.name == "kubeflow/training-operator").newTag) = env(KUBEFLOW_IMAGE_VERSION)' -i "$KUBEFLOW_MANIFEST_MANAGER/kustomization.yaml"
```

Comment:
This cleanup is great to see. Our users will benefit from that simplification of the setup too!